src/plugins/intel_gpu/src/graph/depth_to_space.cpp

```diff
@@ -20,11 +21,6 @@ layout depth_to_space_inst::calc_output_layout(depth_to_space_node const& node,
 
     const size_t block_size = desc->block_size;
 
-    if (block_size < 2)
-        CLDNN_ERROR_MESSAGE(desc->id,
-                            "Invalid depthToSpace block_size value (should equal at least two). Actual block size is" +
-                                std::to_string(block_size));
-
```

Comment:
Why can we remove this assertion?

Reply:
Hmm anyway the spec seems not have such a limitation..

Reply:
Yes, This assertion seems to be a legacy limitation. According to the spec, the default block_size is 1, and I was able to test that case(block_size=1) when I removed this assertion.